scripts/migration/migrate_date_last_request_for_current_users.py

```diff
@@ -0,0 +1,34 @@
+"""
+Used to add users date_last_request, value set equal to date_last_login.
+"""
+
+import sys
+import logging
+from modularodm import Q
+from website.app import init_app
+from website import models
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+
+
+def main(dry=True):
+    init_app(routes=False)
+    users = models.User.find(Q('date_last_request', 'eq', None))
+    logger.warn('All active users will have a date_last_request field added and set equal to date_last_login.')
+
+    for user in users:
+        with TokuTransaction():
+            if user.is_active:
+                user.date_last_request = user.date_last_login
+                if not dry:
```

Comment:
Preferred method of handling dry runs is to run the entire migration in a `TokuTransaction` and raise an error at the end `if dry`. Something like:

```python
def main(dry=True):
    #[perform migration]
    if dry:
        raise RuntimeError('Dry run -- transaction rolled back')

if __name__ == '__main__':
    #[perform setup]
    with TokuTransaction():
        main(dry=dry_run)
```